### PR TITLE
updated a few pointers to addresses

### PR DESCRIPTION
**Pull request by xi-Rick**
the site has some added addresses

### CODE REVIEW

**Review by Slowlife01 — DISMISSED** [review]
Bump version and show proof of it working

File: websites/A/Anime Soul/presence.ts [hunk 1]
```diff
@@ -124,11 +145,23 @@ presence.on("UpdateData", async () => {
...
+  } else if (document.location.pathname.includes("/tasks")) {
+    data.startTimestamp = browsingStamp;
+    data.details = "Viewing Tasks";
   }
   if (data.details == null) {
     presence.setTrayTitle();
     presence.setActivity();
   } else {
     presence.setActivity(data);
   }
```
Comment (EncryptedDev) [inline websites/A/Anime Soul/presence.ts:133]:
```suggestion
presence.setActivity(data)
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Comment by dana0404** [comment]
Screenshot from 2021-09-05 03-13-24

This is display for a newly added address

**Comment by EncryptedDev** [comment]
Look back at my review

**Comment by dana0404** [comment]
why is it this passes the first time

File: websites/A/Anime Soul/presence.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
-var presence = new Presence({
+let presence = new Presence({
```
Comment (EncryptedDev) [inline websites/A/Anime Soul/presence.ts:1]:
```suggestion
const presence = new Presence({
```

File: websites/A/Anime Soul/presence.ts [hunk 3]
```diff
@@ -124,11 +145,24 @@ presence.on("UpdateData", async () => {
...
+  } else if (document.location.pathname.includes("/tasks")) {
+    data.startTimestamp = browsingStamp;
+    data.details = "Viewing Tasks";
   }
   if (data.details == null) {
     presence.setTrayTitle();
     presence.setActivity();
   } else {
     presence.setActivity(data);
   }
```
Comment (EncryptedDev) [inline websites/A/Anime Soul/presence.ts:166]:
```suggestion
    presence.setActivity(data);
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/A/Anime Soul/dist/metadata.json [hunk 4]
```diff
@@ -11,13 +11,13 @@
     "nl": "De grootste Anime Discord server ter wereld!"
   },
   "url": "animesoul.com",
-  "version": "1.1.9",
+  "version": "1.2.0",
   "logo": "https://i.imgur.com/OhVpByX.png",
```
Comment (Slowlife01) [inline websites/A/Anime Soul/dist/metadata.json:15]: Must be at least 512x512

File: websites/A/Anime Soul/presence.ts [hunk 5]
```diff
@@ -40,6 +41,18 @@ presence.on("UpdateData", async () => {
...
+    data.details = "Viewing Articles";
+  } else if (document.location.pathname.includes("/achievements")) {
+    data.startTimestamp = browsingStamp;
+    data.details = "Viewing Achievements";
+  } else if (document.location.pathname.includes("/messages")) {
+    data.startTimestamp = browsingStamp;
+    data.details = "Viewing Messages";
+  } else if (document.location.pathname.includes("/keysgiveaways")) {
+    data.startTimestamp = browsingStamp;
+    data.details = "Viewing Keys Giveaway";
```
Comment (Bas950) [inline websites/A/Anime Soul/presence.ts:55]: just set startTimestamp at the start

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/Anime Soul/presence.ts (hunk 5 above)
Comment (EncryptedDev) [reply websites/A/Anime Soul/presence.ts:55]: then remove curlies

**Comment by EncryptedDev** [comment]
inactivity?

**Pull request closed without merging**